symfit/core/fit.py

```diff
@@ -742,6 +751,11 @@ def _make_signature(self):
         ]
         return inspect_sig.Signature(parameters=parameters)
 
+    def __reduce__(self):
+        return (
+            self.__class__,
+            (self.constraint_type(list(self.values())[0]), self.model)
+        )
```

Comment:
Should this also call `super().__reduce__`? Same goes for all the other reduce methods

Reply:
It could, but I think it would make this way less readable since tuples are immutable and this has to return a tuple of tuples.

Reply:
Ok, do you then actually manage to fully reproduce the object?